utam-core/src/main/java/utam/core/element/Draggable.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.core.element;
+
+/**
+ * interaction methods for an element that can be "dragged"
+ *
+ * @author elizaveta.ivanova
+ * @since 236
+ */
+public interface Draggable extends BasicElement {
```

Comment:
@jimevanssfdc pls check if you're ok with API

Reply:
Looks exactly right to me.